.github/workflows/ci.yml

```diff
@@ -52,6 +52,7 @@ jobs:
           - cp39-cp39
           - cp310-cp310
           - cp311-cp311
+          - cp312-cp312
```

Comment:
do we need to update aws-crt-builder to be sure we're getting cp312 from the official docker images?

Reply:
Thanks, updated.